back/engines/commercial/verification/app/models/verification/verification_method.rb

```diff
@@ -12,5 +12,15 @@ def config
         .symbolize_keys
         .presence
     end
+
+    def fetch_user(request)
+      # `token` can be unusual here. See different `fetch_token` implementations.
+      token = request.env['omniauth.params']['token'] || fetch_token(request)
+      AuthToken::AuthToken.new(token: token).entity_for(::User)
+    end
+
+    protected
+
+    def fetch_token(request); end
```

Comment:
Maybe better to not implement this method at all here? If the used verification method needs this, we probably prefer a hard crash?

Reply:
`AuthToken::AuthToken.new(token: nil).entity_for(::User)` will hardly fail anyway with `ActiveRecord::RecordNotFound: Couldn't find User without an ID`. I did it to preserve the old behavior (for possible error catches and the same Sentry issues). Let me know if you still prefer to raise a specific exception.